nsqd/http.go

```diff
@@ -464,8 +464,14 @@ func (s *httpServer) doStats(w http.ResponseWriter, req *http.Request, ps httpro
 	formatString, _ := reqParams.Get("format")
 	topicName, _ := reqParams.Get("topic")
 	channelName, _ := reqParams.Get("channel")
+	withProducers, _ := reqParams.Get("with_producers")
 	jsonFormat := formatString == "json"
 
+	var producersStats []ClientStats
+	if withProducers == "1" {
```

Comment:
It's sometimes confusing if "0" and "no" and "false" are considered "true", and awkward if you need to override a parameter with a blank string e.g. in your own helper functions. So I humbly suggest checking against "1", "yes", and "true", perhaps case-insensitive.